### PR TITLE
Feature/oo 7173 support formater custom target

### PR DESCRIPTION
**Pull request by jirkapok**

### CODE REVIEW

**Review by findo11 — APPROVED** [review]

File: packages/dashboards/src/lib/configurator/components/formatters/link-formatter/link-formatter.component.ts [hunk 1]
```diff
@@ -15,13 +15,16 @@ export class LinkFormatterComponent implements OnChanges, IHasChangeDetector {
...
+    public targetSelf: string;
     public isValid = false;
 
     constructor(public changeDetector: ChangeDetectorRef) {
     }
 
-    ngOnChanges(changes: SimpleChanges) {
+    public ngOnChanges(changes: SimpleChanges) {
         this.isValid = Boolean(this.data && this.data.value && this.data.link);
+        this.targetSelf = this.target ? "_self" : "_blank";
```
Comment (stepanstava) [inline packages/dashboards/src/lib/configurator/components/formatters/link-formatter/link-formatter.component.ts:28]: do we need to keep it in targetSelf?

we could do:
````
@Input() target = false;
and in template
[target]="target ? '_self' ? '_blank'"
````

File: packages/dashboards/src/lib/configurator/components/widgets/table/columns-editor/column-configuration/presentation-configuration/portals/link-configurator/link-configurator.component.html [hunk 2]
```diff
@@ -31,4 +31,13 @@
             </nui-form-field>
         </div>
     </div>
+    <div>
+        <nui-form-field caption="Open in the same window"
+                        i18n-caption
```
Comment (stepanstava) [inline packages/dashboards/src/lib/configurator/components/widgets/table/columns-editor/column-configuration/presentation-configuration/portals/link-configurator/link-configurator.component.html:36]: i18n-caption is not complete

File: packages/dashboards/src/lib/configurator/components/widgets/table/columns-editor/column-configuration/presentation-configuration/portals/link-configurator/link-configurator.component.html [hunk 3]
```diff
@@ -31,4 +31,13 @@
...
+                        class="mb-3"
+                        [control]="form.controls['target']">
+            <nui-switch id="nui-switch-target" formControlName="target" i18n>
```
Comment (stepanstava) [inline packages/dashboards/src/lib/configurator/components/widgets/table/columns-editor/column-configuration/presentation-configuration/portals/link-configurator/link-configurator.component.html:39]: id seems redundant, i18n too

File: packages/dashboards/src/lib/configurator/components/formatters/link-formatter/link-formatter.component.ts (hunk 1 above)
Comment (jirkapok) [reply packages/dashboards/src/lib/configurator/components/formatters/link-formatter/link-formatter.component.ts:28]: Both have the same behavior, i decided to do it there.

**Review by erzik987 — APPROVED** [review]

**Review by findo11 — APPROVED** [review]

**Pull request merged**